src/InputGroup/__stories__/InputGroup.stories.js

```diff
@@ -1,36 +1,111 @@
 import Button from '../../Button/Button';
+import FormInput from '../../Forms/FormInput';
+import FormTextarea from '../../Forms/FormTextarea';
+import Icon from '../../Icon/Icon';
 import InputGroup from '../InputGroup';
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import {
+    boolean,
     withKnobs
 } from '@storybook/addon-knobs';
 
 storiesOf('Components|InputGroup', module)
     .addDecorator(withKnobs)
     .add('Default', () => (
         <InputGroup>
-            <Button
-                compact
-                glyph='navigation-down-arrow'
-                option='light' />
+            <FormInput />
+            <InputGroup.Addon>
+                <Button
+                    compact
+                    glyph='navigation-down-arrow'
+                    option='light' />
+            </InputGroup.Addon>
```

Comment:
A couple things I noticed:

1. When the `displayName` comparison is fixed in `InputGroup`, these will start working better.
1. Examples that are using buttons need to have the `isButton` prop set on `InputGroup.Addon`.
1. The `compact` prop can be removed from the `Button` components.
1. I see a few of the examples using knobs for the `compact` prop on `InputGroup` -- I think all of these should have that.

Reply:
compact has been added to all example for stories, except for textArea.